modules/python/src2/gen2.py

```diff
@@ -246,9 +247,14 @@ def __init__(self, decl):
 
 class ClassInfo(object):
     def __init__(self, name, decl=None):
+        # Scope name can be a module or other class e.g. cv::SimpleBlobDetector::Params
+        self.scope_name, self.original_name = name.rsplit(".", 1)
+
+        self.export_name = self.original_name
+        self.scope_name = re.sub(r"^cv\.?", "", self.scope_name)
+
+        self.class_id = normalize_class_name(name)
```

Comment:
If we want to rename `.name`, `.wname`, `.cname`, `.sname` wtf fields then we should do it anywhere (for all entities, and for all bindings generators and with clear documentation).

Is it possible to have minimal patch here?

Reply:
Yes, but there is already a mess with the naming, because different parts treats them as they want.
Something uses `wname` as identifier, something uses it as general class name.
At least better naming helps to describe the actual intent.
What is `sname` in this case?

Reply:
I completely agree with that.
My statement that we should not have this change now in the current PR. Lets try to keep in minimal.